KENNEDYMWANGI/API
# #162301750 Create test cases for the app's endpoints.

*What does this PR do?*

Have projects API endpoints (redflags and interventions) test cases set up.

*Descriptions of the tasks to be completed?*

1.Write test cases for the API endpoints.
2.Run the tests using pytest. The tests should fail.

*How should this be manually tested?*

After cloning the repo, cd into it and open terminal and run pytest. All the tests should fail and return 0% coverage.

*Pivotal tracker story*

#162301750
